src/applications/personalization/profile/components/alerts/FieldHasBeenUpdated.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { useLocation } from 'react-router-dom';
+
+export const FieldHasBeenUpdated = () => {
+  const locationState = useLocation().state;
+  const { fieldInfo = null } = locationState || {};
+
+  const text = `We saved your ${fieldInfo?.title?.toLowerCase() ||
+    `information`} to your profile.`;
+
+  return fieldInfo ? (
+    <va-alert
+      class="vads-u-margin-bottom--1"
+      background-only
+      status="success"
+      show-icon
```

Comment:
this is different from the Sketch file, but shouldn't be a big deal since you're using a design system component

Reply:
yep, I mentioned that to Liz as well and she said she will update the sketch file.